src/dao.cpp

```diff
@@ -492,26 +492,60 @@ namespace hypha
 
    bool dao::isPaused() { return false; }
 
+   Document dao::getSettingsDocument(const eosio::name &dao_name)
+   {
+      TRACE_FUNCTION()
+      auto dao = getDAO(dao_name);
+      auto edges = m_documentGraph.getEdgesFromOrFail(dao, common::SETTINGS_EDGE);
+      EOS_CHECK(edges.size() == 1, "There should only exists only 1 settings edge from a dao node");
+      return Document(get_self(), edges[0].to_node);
+   }
+
    Document dao::getSettingsDocument()
    {
       TRACE_FUNCTION()
-      auto root = getRoot(get_self());
+      auto root = getDAO(eosio::name("bm"));
+      // auto root = getRoot(get_self());
```

Comment:
I've high-jacked this function temporarily by hard-coding the ```dao_name```. We'll need to correctly segregate ```dho``` settings from contract settings, but I've done it this way just to get something deployed that will work.

Reply:
TODO: Create the ability to set settings at the contract-level AND at the dho level.